react/src/components/ResourceNumber.tsx

```diff
@@ -0,0 +1,132 @@
+import { iSizeToSize } from '../helper';
+import Flex from './Flex';
+import { Tooltip, Typography } from 'antd';
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+
+type ResourceTypeKey =
+  | 'cpu'
+  | 'mem'
+  | 'cuda.device'
+  | 'cuda.shares'
+  | 'rocm.device'
+  | 'tpu.slot'
+  | 'ipu.slot'
+  | 'atom.slot'
+  | 'warboy.slot'
+  | string;
+interface Props {
+  type: ResourceTypeKey | string;
```

Comment:
`ResourceTypeKey` already implies `string` as a possible type (line 17), so this seems like a duplicated type declaration.

For the future possibility of fractional sharing for other devices, what about using the same convention with the GPUs? For example, `atom.slot` -> `atom.device`.

Reply:
@adrysn 
`ResourceTypeKey` can be  a`string` already. Thank you. 

As `atom.slot`, I don't know why I use it. Haha. I just wanted to follow the [existing code ](https://github.com/lablup/backend.ai-webui/blob/45cd961e30731d930a9d306b4a75a94528df160d/src/components/backend-ai-session-launcher.ts#L1614-L1636). 🤔 It might be due to VSCode Copilot's auto complitions. I'll change.

Reply:
I fixed this in a [recent commit](https://github.com/lablup/backend.ai-webui/pull/1918/commits/73e168febaa8b8017b9e6ab3b8431c6ee1db652a). please check.